services/movie_session.py

```diff
@@ -0,0 +1,47 @@
+from db.models import MovieSession
+
+from django.db.models import QuerySet
+from datetime import datetime
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    query = MovieSession.objects.all()
+    if session_date:
+        date = datetime.strptime(session_date, "%Y-%m-%d")
```

Comment:
Will it work without this `date = datetime.strptime(session_date, "%Y-%m-%d")`?)